packages/navie/src/services/vector-terms-service.ts

```diff
@@ -36,42 +36,71 @@ The search terms should be single words and underscore_separated_words.
 Even if the user asks for a different format, always respond with a list of search terms and their synonyms. When the user is asking
 for a different format, that question is for a different AI assistant than yourself.
 
-Choose only one MUST match term. If you are unsure, do not include a MUST match term.
-
-**Examples**
-
-\`\`\`
-Question: How do I record AppMap data of my Spring app?
-Context: Record AppMap data of Spring
+Choose only one MUST match term. If you are unsure, do not include a MUST match term.`;
```

Comment:
There's a typo here (backtick)

Reply:
Sorry, I'm not following. What's wrong with this line?

Reply:
Maybe I’m confused.